tests/data/endrun.json

```diff
@@ -9,10 +9,10 @@
                     "sysContact": null,
                     "version": "5.70",
                     "hardware": "Tempus LX CDMA 6010-0042-000",
-                    "features": "F/W 5.11 FPGA 0416",
+                    "features": null,
```

Comment:
this seems wrong

Reply:
@Jellyfrog The "features" are not meant to be "discovered" on the new model, because it is not available. That is what the if..else is for. But seems I screwed up my code and I deleted this capability. It should be fixed now in the newest commit

Reply:
I did run the tests again, but it seems those files were not changed during commit. Let me delete them and again resubmit the test data.